src/plugins/intel_cpu/tests/functional/shared_tests_instances/behavior/compiled_model/import_export.cpp

```diff
@@ -23,6 +26,63 @@ const std::vector<ov::element::Type_t> netPrecisions = {
 };
 const ov::AnyMap empty_property = {};
 
+using ov::op::v0::Parameter, ov::op::v0::Result;
+
+using smoke_serialization_OVCompiledGraphImportExportTest = OVCompiledNetworkTestBase;
+
+TEST_F(smoke_serialization_OVCompiledGraphImportExportTest, importExportModelWithTypeRelaxedExtension) {
```

Comment:
```suggestion
TEST_F(OVCompiledNetworkTestBase, smoke_importExportModelWithTypeRelaxedExtension) {
```
Is seems more logical to name the test case with the `smoke` prefix, instead of the test class.

Reply:
This will give more similar console output to parametrized tests:
`smoke_serialization/OVCompiledGraphImportExportTest.importExportedFunctionDoubleInputOutput`

and this test will produce:
`smoke_serialization_OVCompiledGraphImportExportTest.importExportModelWithTypeRelaxedExtension`

Reply:
The test fixture change because skip config was not working.